### PR TITLE
Add bank holidays to the gem

### PR DESCRIPTION
**Pull request by ericaporter**
Adding bank holidays (for England and Wales) to the gem, 2023 - 2025
Bank holidays are taken from [gov.uk](https://www.gov.uk/bank-holidays.json)
Add ability to deploy bank holidays to BigQuery

### CODE REVIEW

File: lib/dfe/reference_data/bank_holidays.rb [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+      BANK_HOLIDAYS_FIELD_DEFINITIONS = {
+        id: 'A unique identifier',
+        title: 'The title of the bank holiday',
+        date: 'The date of the bank holiday',
+        notes: 'Notes about the bank holiday'
+      }.freeze
+
+      # N.B. England and Wales only
+      BANK_HOLIDAYS = DfE::ReferenceData::HardcodedReferenceList.new(
```
Comment (alexchristensenedu) [inline lib/dfe/reference_data/bank_holidays.rb:19]: Please can we also add in 2022 bank holidays?

File: docs/lists_bank_holidays.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+### `DfE::ReferenceData::BankHolidays::BANK_HOLIDAYS`
+
+Bank Holidays.
+
+Owner: None.
+
+Users: None.
+
+Source: https://www.gov.uk/what-different-qualification-levels-mean/list-of-qualification-levels
```
Comment (asatwal) [inline docs/lists_bank_holidays.md:17]: Is this a Copy+Paste error ?This source seems to be related to qualifications.

**Review by asatwal — COMMENTED** [review]
Looks OK to me. Just a questions about the docs.

File: lib/dfe/reference_data/bank_holidays.rb (hunk 1 above)
Comment (ericaporter) [reply lib/dfe/reference_data/bank_holidays.rb:19]: Now added 👍

File: docs/lists_bank_holidays.md (hunk 2 above)
Comment (ericaporter) [reply docs/lists_bank_holidays.md]: Good spot! Now updated!

**Pull request merged**